using_images/other_images/jenkins.adoc

```diff
@@ -417,3 +417,29 @@ Jenkins job] that is pre-configured in the Jenkins image utilizes the
 plug-in for creating CI/CD flows for {product-title} in Jenkins.
 
 See the https://github.com/openshift/jenkins-plugin/[the plug-in's README] for a detailed description of what is available.
+
+== OpenShift Client Plug-in
+
+The rapid evolution of both Jenkins (including Pipelines and the introduction of Global Tool
+Configuration in Jenkins V2) and OpenShift have had an effect on how to best provide OpenShift integration from
+Jenkins jobs.  The advantages of not requiring the `oc` binary to be included in the Jenkins path, a key design point
+for OpenShift Pipeline plug-in, have been largely mitigated.  And the lack of functionality with client
+side logic in addition to REST flows, such as seen with `oc new-app` or binary builds, is more of a detriment as usage
+of the OpenShift Pipeline plug-in has expanded.
+
+Wit these realizations in mind, the new experimental https://github.com/openshift/jenkins-client-plugin[OpenShift Client Plugin for Jenkins]
```

Comment:
s/Wit/With/ but i think based on my comment above this whole sentence is likely to change.